images/hub/requirements.txt

```
@@ -109,15 +109,15 @@ markupsafe==2.1.5
     # via
     #   jinja2
     #   mako
-multidict==6.0.5
+multidict==6.1.0
     # via
     #   aiohttp
     #   yarl
 mwoauth==0.4.0
     # via -r requirements.in
 nullauthenticator==1.0.0
     # via -r requirements.in
-oauthenticator==16.3.1
+oauthenticator==17.0.0
```

Opening a dedicated PR about this.

```suggestion
oauthenticator==16.3.0
```